README.md

```diff
@@ -70,13 +103,21 @@ TensorFlow [prepare environment] for linux.
    source <your_virtual_env_dir>/bin/activate # bash, sh, ksh, or zsh
    ```
 
-3. Now run `./configure` and choose all the defaults when prompted to build TensorFlow.
+3. Now run `./configure` and choose `no` for all the questions when prompted to build TensorFlow.
```

Comment:
Should we change it to 'choose configure options as you like' .. or something on those lines. .. ?